src/sentry/middleware/health.py

```diff
@@ -19,9 +21,16 @@ def process_request(self, request):
 
         from sentry.status_checks import check_all
         from sentry.utils import json
-        problems, checks = check_all()
 
+        results = check_all()
```

Comment:
Right. Otherwise now all of a sudden this will return a 500 (thus fail pingdom checks) if any warnings happen.